server/views/pages/allocation-dashboard/allocation-dashboard.njk

```diff
@@ -177,7 +177,7 @@
                     }
                 },
                 {
-                    text: row.startDate | formatDate('d MMMM yyyy') + (' to ' + row.endDate | subDays(1) |formatDate('d MMMM yyyy') if row.endDate)
```

Comment:
After discussing the sub one day from the allocation end date with the BA's it was decided this would be confusing ergo it has been removed.